alerts.tf

```diff
@@ -54,7 +54,7 @@ resource "newrelic_nrql_alert_condition" "error_rate" {
 
   nrql {
     query = <<-EOF
-        SELECT percentage(count(*), WHERE error IS TRUE)
+        SELECT 100 * (filter(count(*), WHERE error IS TRUE)/(count(*) + 1e-10)) as Percentage
```

Comment:
Is there any open issue related to this workaround? If so, it would be nice to add the link here (as well as in the other occurrences)

Reply:
No, as far as I know there is no open issue related to that. It looks like it's the way NewRelic works.

Reply:
Then let's add a comment why we do this crazy thing.